style.css

```diff
@@ -0,0 +1,165 @@
+@charset "UTF-8";
+
+
+
+body{
```

Comment:
Sempre use um espaço entre o seletor (body) e o colchete!
```suggestion
body {
```

Reply:
Obrigado